src/adhocracy_mercator/adhocracy_mercator/scripts/export_users.py

```diff
@@ -7,16 +7,61 @@
 import sys
 import textwrap
 import time
-
 from pyramid.paster import bootstrap
 from substanced.util import find_catalog
 
+from adhocracy_core.catalog.adhocracy import index_rates
+from adhocracy_core.resources.principal import IUser
+from adhocracy_core.sheets.metadata import IMetadata
 from adhocracy_core.sheets.principal import IUserBasic
 from adhocracy_core.sheets.principal import IUserExtended
-from adhocracy_core.sheets.metadata import IMetadata
+from adhocracy_core.sheets.rate import IRateable
+from adhocracy_core.sheets.rate import IRate
 from adhocracy_core.utils import get_sheet_field
+from adhocracy_mercator.resources.mercator import IMercatorProposalVersion
+from adhocracy_mercator.sheets.mercator import ITitle
+from adhocracy_core.sheets.tags import filter_by_tag
+
+
+def _get_most_rated_proposals(root, min_rate=100):
+    catalog = find_catalog(root, 'system')
+    adhocracy_catalog = find_catalog(root, 'adhocracy')
+    path = catalog['path']
+    interfaces = catalog['interfaces']
+    tag = adhocracy_catalog['tag']
+
+    query = path.eq('/mercator') \
+        & interfaces.eq(IMercatorProposalVersion) \
+        & tag.eq('LAST')
+
+    proposals = query.execute()
+    proposals = [p for p in proposals if index_rates(p, None) > min_rate]
+    proposals = list(reversed(sorted(proposals,
```

Comment:
performance: What we need here is a search for proposals aggregatedby  `rates` index. 
The current filtered pool sheet implementation does this already, but only returns the count number for each `rate` index value